test/runtests.jl

```diff
@@ -62,12 +63,8 @@ end
         τ_strat = τ_strat, wm = similar(vs)
     ).df
     r = autoblock(rdfs, start=101)
-    #@test reduce(&, Tuple(r).≈(-5.25223493152101, 0.19342756375228998, -6.527599639255635, 0.5197276766889821, 6))
-    if OV
-        @test reduce(&, Tuple(r).≈(-5.25223493152101, 0.19342756375229, -6.527599639255635, 0.47607219512729554, 6))
-    else
-        @test reduce(&, Tuple(r).≈(-5.714600548611788, 0.21631081209341332, -5.884807394477632, 0.3849918114544903, 6))
-    end
+    @test reduce(&, Tuple(r).≈(-5.459498724286854, 0.19512207462981085, -6.474791532530609, 0.5301321595206501, 6))
```

Comment:
You could use `all` instead of `reduce`.

Reply:
done